src/box/memtx_tx.c

```diff
@@ -1499,7 +1499,8 @@ check_hole(struct space *space, uint32_t index,
 
```

Comment:
Could you please make the title of the commit a bit shorter? It is shown very strange in Web (an I bet somewhere else).

Reply:
Fixed: shortened the commit subject.

@locker seems like a bug in checkpatch (tarantool/checkpatch#18).